docs/src/prism/prism-oly.js

```diff
@@ -0,0 +1,28 @@
+Prism.languages.mew = Prism.languages.extend('clike', {
+	'string': {
+		pattern: /(^|[^\\])"(?:\\.|[^"\\\r\n])*"|`[^`]*`/,
+		lookbehind: true,
+		greedy: true
+	},
+	'keyword': /\b(?:union|where|namespace|match|override|ctor|base|prop|mut|property|constructor|static|class|pub|module|use|enum|type|trait|interface|impl|field|new|while|loop|break|continue|else|fn|if|return|let|for|in|as|print)\b/,
```

Comment:
Looks like this was borrowed from Mew. If you want to see the list of a keywords, the source of truth is here: https://github.com/TIHan/Oly/blob/main/src/Compiler/Oly.Compiler.Syntax/Internal/Token.fs